packages/core/src/components/table/table-component-basic.stories.tsx

```diff
@@ -233,91 +245,49 @@ const BasicTemplate = ({
-    cellTextAlignment ? `text-align="${cellTextAlignment}"` : ''
-  }></tds-body-cell>
-              <tds-body-cell cell-value="Test value 3" cell-key="country" disable-padding="${disablePadding}" ${
-    cellTextAlignment ? `text-align="${cellTextAlignment}"` : ''
-  }></tds-body-cell>
-              <tds-body-cell cell-value="Test value 4" cell-key="mileage" disable-padding="${disablePadding}" ${
-    cellTextAlignment ? `text-align="${cellTextAlignment}"` : ''
-  }></tds-body-cell>
-          </tds-table-body-row>
-          <tds-table-body-row>
-              <tds-body-cell cell-value="Test value 5" cell-key="truck" disable-padding="${disablePadding}" ${
-    cellTextAlignment ? `text-align="${cellTextAlignment}"` : ''
-  }></tds-body-cell>
-              <tds-body-cell cell-value="Test value 6" cell-key="driver" disable-padding="${disablePadding}" ${
-    cellTextAlignment ? `text-align="${cellTextAlignment}"` : ''
-  }></tds-body-cell>
-              <tds-body-cell cell-value="Test value 7" cell-key="country" disable-padding="${disablePadding}" ${
-    cellTextAlignment ? `text-align="${cellTextAlignment}"` : ''
-  }></tds-body-cell>
-              <tds-body-cell cell-value="Test value 8" cell-key="mileage" disable-padding="${disablePadding}" ${
-    cellTextAlignment ? `text-align="${cellTextAlignment}"` : ''
-  }></tds-body-cell>
-          </tds-table-body-row>
+          ${[...Array(6)]
+            .map(
+              (_, index) => `
+            <tds-table-body-row ${clickable ? 'clickable' : ''}>
+              <tds-body-cell cell-value="Test value ${
+                index % 2 === 0 ? '1' : '5'
+              }" cell-key="truck" disable-padding="${disablePadding}" ${
+                cellTextAlignment ? `text-align="${cellTextAlignment}"` : ''
+              }></tds-body-cell>
+              <tds-body-cell cell-value="Test value ${
+                index % 2 === 0 ? '2' : '6'
+              }" cell-key="driver" disable-padding="${disablePadding}" ${
+                cellTextAlignment ? `text-align="${cellTextAlignment}"` : ''
+              }></tds-body-cell>
+              <tds-body-cell cell-value="Test value ${
+                index % 2 === 0 ? '3' : '7'
+              }" cell-key="country" disable-padding="${disablePadding}" ${
+                cellTextAlignment ? `text-align="${cellTextAlignment}"` : ''
+              }></tds-body-cell>
+              <tds-body-cell cell-value="Test value ${
+                index % 2 === 0 ? '4' : '8'
+              }" cell-key="mileage" disable-padding="${disablePadding}" ${
+                cellTextAlignment ? `text-align="${cellTextAlignment}"` : ''
+              }></tds-body-cell>
+            </tds-table-body-row>
+          `,
+            )
+            .join('')}
       </tds-table-body>
-  </tds-table>`);
+  </tds-table>
+
+  <!-- The script below is just for demo purposes -->
+  <script>
+    // Select all rows
+    const rows = document.querySelectorAll('tds-table-body-row');
```

Comment:
Should be a let since it need to reset on each screen load. I think this is where your error is coming from @theJohnnyMe